common/djangoapps/util/tests/test_milestones_helpers.py

```diff
@@ -4,6 +4,11 @@
 
 from mock import patch
 from util import milestones_helpers
+
+from milestones.exceptions import (
```

Comment:
Good catch. Initially I thought I was going to assertRaise them but then I realized I just returned False. will remove.